main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        numberSelector
+//  action 1:    enumerates even numbers
+numberSelector.enumerateEven();
+//  action 2:    checks 0 presence
+numberSelector.checkZeroPresence();
+//  action 3:    applies an inversion on an array
+numberSelector.invertArray();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        oscillator
+//  action 1:    produces sine
+oscillator.produceSine();
+//  action 2:    adds a limit threshold
+oscillator.setLimitThreshold();
+//  action 3:    modulates sine with Square
+oscillator.modulateSineWith();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.